tools/nl/embeddings/data/finetuning/sentence_pairs.csv

```diff
@@ -1,32 +1,118 @@
 sentence_1,sentence_2,score
```

Comment:
nit:  wonder if its useful to allow and add comments?  (maybe follow on PR)

Reply:
Yup, it is on my list of TODOs